ci/prow/config.yaml

```diff
@@ -1739,6 +1753,20 @@ periodics:
       - "./hack/release.sh"
       - "--publish"
       - "--tag-release"
+- cron: "15 9 * * *" # Run at 02:15PST every day (09:15 UTC)
+  name: ci-knative-backup-artifacts
+  agent: kubernetes
+  labels:
+    preset-backup-account: "true"
+  spec:
+    containers:
+    - image: gcr.io/knative-tests/test-infra/backups:latest
+      imagePullPolicy: Always
+      command:
+      - "/backup.sh"
+      resources:
```

Comment:
Just curious, we really need 1G for this job?

Reply:
Almost certainly not, but I'm not sure how to right-size it, and I'm not sure how much memory gcloud/gsutil consumes. Any pointers?

Reply:
Sorry, no. Let's have it run for a while, then we can inspect past runs and tweak it, if necessary.